src/main/java/com/thebois/views/toolbar/ToolbarView.java

```diff
@@ -0,0 +1,33 @@
+package com.thebois.views.toolbar;
+
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.ui.HorizontalGroup;
+
+import com.thebois.views.info.IActorView;
+
+/**
+ * The view that displays the different items in the toolbar.
+ */
+public class ToolbarView {
+
+    private final HorizontalGroup toolbar;
+
+    /**
+     * Instantiates a new view of a toolbar that the player can interact with.
+     *
+     * @param widgetViews The items that should be displayed in the toolbar.
+     */
+    public ToolbarView(final Iterable<IActorView> widgetViews) {
+
+        toolbar = new HorizontalGroup();
+        toolbar.expand().fill().expand();
+        for (final IActorView actorView : widgetViews) {
+            toolbar.addActor(actorView.getWidgetContainer());
+        }
+    }
+
+    public Actor getPane() {
```

Comment:
Missing `@Override`

Reply:
???
Toolbar view doesn't implement anything, just like info view.

Reply:
They both really should... Guess we'll have to write that in the SDD, then.